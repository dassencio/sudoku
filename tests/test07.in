2 5
 _  7  _  _  _  _  8  _  _ 10
 _  _  5  _  _  4  _  _  1  _
 _  _  7  9  _  _  _  4  _  3
 _  _  3  _  _  8  9  _  5  7
 6  _  _  7  _  _  _  8 10  _
 _ 10  8  _  _  _  2  _  _  6
 9  3  _  4  8  _  _  7  _  _
10  _  2  _  _  _  1  9  _  _
 _  5  _  _  6  _  _  1  _  _
 3  _  _  2  _  _  _  _  7  _
