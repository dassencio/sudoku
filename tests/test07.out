 2  7  6  1  4  5  8  3  9 10
 8  9  5 10  3  4  7  6  1  2
 5  8  7  9  2  1 10  4  6  3
 4  1  3  6 10  8  9  2  5  7
 6  2  4  7  5  9  3  8 10  1
 1 10  8  3  9  7  2  5  4  6
 9  3  1  4  8 10  6  7  2  5
10  6  2  5  7  3  1  9  8  4
 7  5 10  8  6  2  4  1  3  9
 3  4  9  2  1  6  5 10  7  8
